wiki/Contests/M2P/en.md

```diff
@@ -0,0 +1,103 @@
+
+## Previous iterations
+
+| Contest No. | Judge | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Results |
+| :-: | :-- | :-- | :-- | :-- | :-- |
+| [#1](https://osu.ppy.sh/community/forums/topics/1624880) | ::{FLAG=NL}:: [Boaz](https://osu.ppy.sh/users/13302996) | ::{FLAG=JP}:: [_Rise](https://osu.ppy.sh/users/5217107) | ::{FLAG=CL}:: [Metsaller](https://osu.ppy.sh/users/4364791) | ::{FLAG=FI}:: [Antti](https://osu.ppy.sh/users/13281473) | [Results](https://docs.google.com/spreadsheets/d/1aR1GjF8diyjFhK_k_uOkU40zIKjaQbXhVbGDGmq5pdg/edit?usp=sharing) |
+| [#2](https://osu.ppy.sh/community/forums/topics/1639586) | ::{FLAG=AR}:: [Axer](https://osu.ppy.sh/users/7299864) | ::{FLAG=TH}:: [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) | ::{ flag=SG }:: [\_gt](https://osu.ppy.sh/users/8301957) | ::{FLAG=US}:: [Vixlowo](https://osu.ppy.sh/users/20295114) | [Results](https://docs.google.com/spreadsheets/d/1ps3bOL2JlzwPEh0gVTLHnp7E48stwqHMH6-ljDSBJQs/edit?usp=sharing) |
+| [#3](https://osu.ppy.sh/community/forums/topics/1652502) | ::{FLAG=US}:: [radar](https://osu.ppy.sh/users/7131099) | *TBD* | *TBD* | *TBD* | *TBD* |
+
+## Rewards
+
+The Mapping to Pander: Taiko contest series rewards its winners with the following prizes:
+
+- **osu!supporter rewards:** Each iteration can have a different amount of osu!supporter to award the winner, or the top 3 in some occasions.
+- **Ranked status:** ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) and another Beatmap Nominator will check and nominate the winning entry if the mapper wants to, and **if they get the necessary difficulties to form a rankable spread when needed.**
+- **Pander Points:** Each iteration will award points to the top 3 contestants. The points are awarded as follows:
+  - **1st place:** 3 points
+  - **2nd place:** 2 points
+  - **3rd place:** 1 point
+
+Pander Points reward users with a **profile badge**, which can be upgraded at certain thresholds:
+
+| Points | Badge |
+| :-: | :-- |
+| 9 | ![M2P Tier 1](img/t1.png "M2P: Elite Panderer - Tier 1") |
+| 18 | ![M2P Tier 2](img/t2.png "M2P: Elite Panderer - Tier 2") |
+| 27 | ![M2P Tier 3](img/t3.png "M2P: Elite Panderer - Tier 3") |
+
+## Pander Points leaderboard
+
+| User | Points |
+| :-- | :-: |
+| ::{ flag=JP }:: [_Rise](https://osu.ppy.sh/users/5217107) | 3 |
+| ::{ flag=TH }:: [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) | 3 |
+| ::{ flag=CL }:: [Metsaller](https://osu.ppy.sh/users/4364791) | 2 |
+| ::{ flag=SG }:: [\_gt](https://osu.ppy.sh/users/8301957) | 2 |
+| ::{ flag=FI }:: [Antti](https://osu.ppy.sh/users/13281473) | 1 |
+| ::{ flag=US }:: [Vixlowo](https://osu.ppy.sh/users/20295114) | 1 |
+
+## Notes
+
+[^schedules-screening]: The screening phase usually depends on the amount of entries submitted, and is not always necessary. If present, it can last for 1 to 3 days.
+
+[^schedules-judging]: The judging period usually depends on the availability of the judge.
```

Comment:
no need for an empty line
```suggestion
[^schedules-screening]: The screening phase usually depends on the amount of entries submitted, and is not always necessary. If present, it can last for 1 to 3 days.
[^schedules-judging]: The judging period usually depends on the availability of the judge.
```